composer.json

```diff
@@ -94,8 +95,8 @@
         "thirdparty/"
     ],
     "scripts": {
-        "lint": "phpcs src/ tests/php",
-        "lint-clean": "phpcbf src/ tests/php",
+        "lint": "vendor/bin/phpcs src/ tests/php",
```

Comment:
This isn't going to work if framework isn't root, meaning these scripts are no longer useful to devs.